src/util.cpp

```diff
@@ -58,6 +58,7 @@
 #ifndef NOMINMAX
 #define NOMINMAX
 #endif
+#include <codecvt>
```

Comment:
maybe include only if `WIN32`?

Reply:
OK. But I'll wait for gitian build done.

Reply:
This is already be defined only if WIN32. Forgot about this

Reply:
Previously applied patch detected.

Needs rebase?

Reply:
Rebased